dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi3.0-appSecurity.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.cdi3.0-appSecurity
+visibility=private
+IBM-Provision-Capability: \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-3.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.appSecurity-4.0))"
+-bundles=\
+ com.ibm.ws.cdi.security
```

Comment:
does `com.ibm.ws.cdi.security` need to be transformed?

Reply:
No. It does not depend on any Java EE classes.